partner_network_connect.go

```diff
@@ -49,6 +49,8 @@ type PartnerAttachmentCreateRequest struct {
 	BGP BGP `json:"bgp,omitempty"`
 	// RedundancyZone is the redundancy zone of the Partner Attachment
 	RedundancyZone string `json:"redundancy_zone,omitempty"`
+	// Represents the uuid of the parent Partner Attachment if there is one
+	ParentUuid string `json:"parent_uuid,omitempty"`
```

Comment:
nit: The comments for these fields should start with the field name. e.g. `ParentUuid represents the uuid of the parent Partner Attachment if there is one`